src/coreclr/vm/interoplibinterface_comwrappers.cpp

```diff
@@ -614,4 +614,33 @@ extern "C" CLR_BOOL QCALLTYPE TrackerObjectManager_IsGlobalPeggingEnabled()
     return InteropLibImports::GetGlobalPeggingState();
 }
 
+// Some of our "Untracked" COM objects may be owned by static globals
+// in client code. We need to ensure that we don't try executing a managed
+// method for the first time when the process is shutting down.
+// Therefore, we need to provide unmanaged implementations of AddRef and Release.
+namespace
+{
+    int STDMETHODCALLTYPE Untracked_AddRef(void*)
+    {
+        return 1;
+    }
+    int STDMETHODCALLTYPE Untracked_Release(void*)
+    {
+        return 1;
+    }
+}
+
+extern "C" void* QCALLTYPE ComWrappers_GetUntrackedAddRef()
```

Comment:
```suggestion
extern "C" void* QCALLTYPE ComWrappers_GetUntrackedAddRefRelease()
```
Nit: The implementations for both AddRef and Release are exactly the same. We can have one method and reuse it for both.